.gitignore

```diff
@@ -17,3 +17,4 @@ narayana-tmp/
 benchmark-output.txt
 benchmark.png
 bm-output.txt
+.metadata*
```

Comment:
Please can you clarify why this is added?

Reply:
This is added in order to ignore all .metadata files created by the IDE (i.e. log, index and location files). I thought it might be helpful also for other developers.

Reply:
Thank you